test_regress/t/t_dist_cppstyle.pl

```diff
@@ -9,7 +9,12 @@
 # SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
 
 scenarios(dist => 1);
+skip("CentOS hangs on this test ")
+  if (1);
 
+
+ok(1);
+1;
```

Comment:
Don't commit this, or check for appropriate clang version that is different and make a separate pull please.

Reply:
Apologies, local mess - will clean up. Thanks for the review